build.gradle.kts

```diff
@@ -89,45 +53,13 @@ tasks {
             kotlinOptions.jvmTarget = "1.8"
         }
     }
-
-    withType<Detekt> {
-        jvmTarget = "1.8"
-    }
-
     patchPluginXml {
-        version(pluginVersion)
-        sinceBuild(pluginSinceBuild)
-        untilBuild(pluginUntilBuild)
-
-        // Extract the <!-- Plugin description --> section from README.md and provide for the plugin's manifest
-        pluginDescription(
-            closure {
-                File("./README.md").readText().lines().run {
-                    val start = "<!-- Plugin description -->"
-                    val end = "<!-- Plugin description end -->"
-
-                    if (!containsAll(listOf(start, end))) {
-                        throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
-                    }
-                    subList(indexOf(start) + 1, indexOf(end))
-                }.joinToString("\n").run { markdownToHTML(this) }
-            }
-        )
-
-        // Get the latest available change notes from the changelog file
-        changeNotes(
-            closure {
-                changelog.getLatest().toHTML()
```

Comment:
This is the logic that was reading the CHANGELOG.md and adding it to the plugin description. I got rid of it entirely, as it turned out the "Changelog" field in the plugin.xml is not mandatory. This means we will not have to maintain a CHANGELOG.md any more and there will be no changelog information in the marketplace any more. 

I can live with that because by that I can get rid of a lot of voodoo in the build process.
I don't think a CHANGELOG.md adds any value in our simple release process. Release Notes will still be available in the git release-tags.